Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby "2.2.10"
+ruby "2.3.7"
```

Comment:
I'm wondering if we should remove this line altogether. I believe that the purpose is to make Bundler fail fast if the used Ruby version isn't right. [Docs](https://bundler.io/v1.7/gemfile_ruby.html). But our application does work with several Ruby versions at the moment (2.1, 2.2, 2.3). So I would skip this requirement and see the `.ruby-version` file as source of truth to select the best version.

Reply:
yes, I think we'll get to it as soon as it breaks our development with the two versions.